install/kubernetes/cilium/templates/_helpers.tpl

```diff
@@ -9,9 +9,9 @@ Create chart name and version as used by the chart label.
 Return the appropriate apiVersion for ingress.
 */}}
 {{- define "ingress.apiVersion" -}}
-{{- if semverCompare ">=1.4-0, <1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- if semverCompare ">=1.4-0, <1.19-0" .Capabilities.KubeVersion.Version -}}
```

Comment:
good pick on version 1.14 here :100: . For other reviewer, Ingress in networking.k8s.io/v1beta1 is available from from 1.14, but Integress is available in networking.k8s.io/v1 from 1.19+ only.

Reply:
So where is the `networking.k8s.io/v1beta1`? I see `extensions/v1beta1 and `networking.k8s.io/v1` but not `networking.k8s.io/v1beta1`.

Reply:
@aanm if I understand correctly you want to use latest api version in relation to each Kubernetes version?
Right now this works fine, but if needed we can have better versioning.

Reply:
@aanm I've introduced `networking.k8s.io/v1beta1` starting from 1.14 as [documented here](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/) 
Let me know if there is anything else.